src/app.js

```diff
@@ -1,29 +1,29 @@
-const path = require("node:path");
-const express = require("express");
+const path = require('node:path');
+const express = require('express');
 const app = express();
-const cors = require("cors");
-const favicon = require("express-favicon");
-const logger = require("morgan");
+const cors = require('cors');
+const favicon = require('express-favicon');
+const logger = require('morgan');
 
 // imports
-const testsRouter = require("./routes/testsRouter");
-const activitiesRouter = require("./routes/activitiesRouter");
+const testsRouter = require('./routes/testsRouter');
+const activitiesRouter = require('./routes/activitiesRouter');
 
-const { errorHandler, notFound } = require("./middleware/errorHandler");
+const { errorHandler, notFound } = require('./middleware/errorHandler');
 
 // middleware
 
 // we shall change the cors origin once the frontend is deployed
-app.use(cors({ origin: process.env.ORIGIN, optionsSuccessStatus: 200 }));
+app.use(cors({ origin: process.env.ORIGIN || "http://localhost:3000", optionsSuccessStatus: 200 }));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(logger("dev"));
-app.use(express.static("public"));
-app.use(favicon(path.join(__dirname, "/public/favicon.ico")));
+app.use(logger('dev'));
+app.use(express.static('public'));
+app.use(favicon(path.join(__dirname, '/public/favicon.ico')));
 
 // routes
-app.use("/api/v1", testsRouter);
-app.use("/api/v1/activities", activitiesRouter);
+app.use('/api/v1', testsRouter);
+app.use('/api/v1/activities', activitiesRouter);
```

Comment:
@catbus00 would you mind removing the other changes you've made that don't involve your update on line 17? (I believe @msrezaie has prettier set to prefer double quotes.)

Since this is such a small change, I'd suggest just "throwing away" this commit, then pulling an updated copy of this repo, then making your change. To simplify this, you can even start a new branch and open a new PR and we can close this one when you've done that.

```
git checkout dev
git pull origin dev
git checkout -b <your-new-branch>
```

Then make your change. You can check the modifications you have with `git status` and if there are changes other than what you want to change on line 17, you can stage just your change by staging in hunks with this

```
git add -p src/app.js
```
Git will then run through each change in that file and ask if you'd like to stage it or not. Once you're done staging, then you can commit only the staged change with `git commit -m "your commit msg"`

Reply:
I'm sorry, I think these extra changes are happening because the backend repo doesn't have the config it needs for Prettier. Before it was ok because it was only Mo pushing to this repo, but I think we've got some different settings happening between different people's IDEs

https://github.com/Code-the-Dream-School/ee-prac-team2-back/issues/43

Reply:
@dargaCode yes, I didn't add any other changes manually - just this one `app.use(cors({ origin: process.env.ORIGIN || "http://localhost:3000",`.